src/Maker/StoryRenderer.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Storybook\Maker;
+
+use Symfony\Bundle\MakerBundle\Generator;
+use Symfony\UX\TwigComponent\ComponentFactory;
+
+class StoryRenderer
```

Comment:
Is it possible to rename this class to something that doesn't collide with the main `StoryRenderer`? Like `StoryGenerator`?

Also this class could be final